code/modules/vending/_vending.dm

```diff
@@ -992,7 +992,8 @@
 			flick(icon_deny,src)
 			vend_ready = TRUE
 			return
-		else if(!C.registered_account.account_job)
+		// Department cards cannot be used to order stuff in vendors, we make an exception for the debug card
+		else if(!C.registered_account.account_job && !istype(C, /obj/item/card/id/syndicate/debug))
```

Comment:
Could this instead be a flag on the card rather than a job check, just to prevent checking a specific type in generic code and cleaning it up (or alternatively assign a debug job to the debug card)?